README.md

```diff
@@ -588,13 +593,13 @@ Finally, you might not need this GH Action at all. For example, if you want to s
 ### More examples
 
 <!-- prettier-ignore-start -->
-Name | Description
---- | ---
-[cypress-gh-action-example](https://github.com/bahmutov/cypress-gh-action-example) | uses Yarn, and runs in parallel on several versions of Node, also different browsers
-[cypress-gh-action-monorepo](https://github.com/bahmutov/cypress-gh-action-monorepo) | splits install and running tests commands, runs Cypress from sub-folder
-[cypress-gh-action-subfolders](https://github.com/bahmutov/cypress-gh-action-subfolders) | separate folder for Cypress dependencies
-[cypress-gh-action-split-install](https://github.com/bahmutov/cypress-gh-action-split-install) | only install NPM dependencies, then install and cache Cypress binary yourself
-[gh-action-and-gh-integration](https://github.com/cypress-io/gh-action-and-gh-integration) | records to the dashboard and has [Cypress GH Integration](https://on.cypress.io/github-integration) app installed
+| Name                                                                                           | Description                                                                                                       |
```

Comment:
why did it reformat the code here - if I have Prettier ignore comments around it? Could you revert this change, I really do not like those padded tables

Reply:
Sure thing! doing it now